modules/oe_showcase_multilingual/tests/src/ExistingSite/WebToolsEtransTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_showcase_multilingual\ExistingSiteJavascript;
+
+use Drupal\Tests\oe_showcase\ExistingSite\ShowcaseExistingSiteTestBase;
+
+/**
+ * Tests oe_webtools_etrans block is displaying properly.
+ */
+class WebToolsEtransTest extends ShowcaseExistingSiteTestBase {
+
+  /**
+   * Test oe_webtools_etrans block in oe_showcase.
+   */
+  public function testWebToolsEtrans(): void {
+    $this->markEntityTypeForCleanup('node');
+
+    $assert_session = $this->assertSession();
+    $page = $this->getSession()->getPage();
+
+    $user = $this->createUser([
+      'create oe_showcase_page content',
+      'delete own oe_showcase_page content',
+      'edit own oe_showcase_page content',
+    ]);
+    $this->drupalLogin($user);
+
+    // Assert Etrans is loaded for nodes with no translations.
+    $values = [
+      'type' => 'oe_showcase_page',
+      'title' => 'Test page',
+      'body' => 'This profile is aimed to provide developers with a series of examples on how the OpenEuropa Library features are built and configured.',
+      'status' => 1,
+      'language' => 'en',
+    ];
+    $node = $this->createNode($values);
+    $this->drupalGet($node->toUrl());
+
+    $etrans_block = $page->find('css', 'main > div.container > div.row > div.col-12 > div#block-showcase-etrans');
```

Comment:
```suggestion
    $etrans_block = $assert_session->elementExists('css', 'main > div.container > div.row > div.col-12 > div#block-showcase-etrans');
```
So we can drop the $page too.